FlowCal/excel_ui.py

```diff
@@ -181,16 +188,16 @@ def process_beads_table(beads_table,
     mef_transform_fxns : OrderedDict
         A dictionary of MEF transformation functions, indexed by
         ``beads_table.index``.
-    mef_outputs : OrderedDict
-        A dictionary with the intermediate results of the generation of the
-        MEF transformation functions, indexed by ``beads_table.index``.
-        Only included if `full_output` is True.
+    mef_outputs : list
+        A list with intermediate results of the generation of the MEF
+        transformation functions, indexed by ``beads_table.index``. Only
+        included if `full_output` is True.
```

Comment:
what was the reason for changing this to a list? I'm not against it, just wondering if there was a motivating reason.